### ISSUE TITLE
Is there any intention to support also the cudadecoder in this gstreamer setting ?

### ISSUE THREAD

**Issue by RaniRaven** [alumae/gst-kaldi-nnet2-online#91]
It seems like there is no server at all supporting the cudadecoder, and this is the only thing which supports GPU now.
It's a pitty, since the result could have been much faster with lower latency.

**Comment by alumae** [comment]
No, there are no plans to support CUDA. I just cannot see it as a very practical approach (one would need one GPU per worker, which would be expensive, and CPU decoding is already faster than real time in most cases).
